docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md

```diff
@@ -0,0 +1,134 @@
+
+  - `handler.js` with the Function's code and the simple "Hello Serverless" logic
+  
+  - `package.json` with the Function's dependencies
+
+4. In the `config.yaml` file configure an API Rule to expose your Function to the incoming traffic over HTTP. Enter the subdomain name as the `host` property:
+
+    ```yaml
+    apiRules:
+      - name: incoming-http-trigger
+        service:
+          host: incoming
+        rules:
+          - methods:
+              - GET
+            accessStrategies:
+              - handler: allow
+    ```
+
+5. Provide your Function logic in the `handler.js` file. In the following example you do not find an actual sanitisation logic, `sanitise` Function is just a placeholder:
+
+   ```js
+   const { v4: uuidv4 } = require('uuid');
+   module.exports = {
+       main: function (event, context) {
+           let sanitsedData = sanitise(event.data)
+           var eventOut=event.buildResponseCloudEvent(uuidv4(), "sap.kyma.custom.acme.payload.sanitised.v1", sanitisedData);
+           eventOut.source="kyma"
+           eventOut.specversion="1.0"
+           event.publishCloudEvent(eventOut);
+           console.log(`Payload pushed to sap.kyma.custom.acme.payload.sanitised.v1`,eventOut)
+           return eventOut;
+       }
+   }
+   let sanitise = (data)=>{
+       console.log(`sanitising data...`)
+       console.log(data)
+       return data
+   }
+   ```
+   The `sap.kyma.custom.acme.payload.sanitised.v1` is an exemplary event type declared by the emitter Function when publishing events. You can choose a different one that better suits your use case. Keep in mind the constraints described on the [Event names](https://kyma-project.io/docs/kyma/latest/05-technical-reference/evnt-01-event-names) page. The receiver subscribes to the event type to consume the events.
+
+   Please note that event object provides convinience functions to build and publish events. To send the event you need to build the Cloud Event. To learn more please visit the [Function's Specification](https://kyma-project.io/docs/kyma/latest/05-technical-reference/svls-08-function-specification#event-object-sdk) page. In addition, your `eventOut.source` key needs to point at `“kyma”` to use Kyma In-cluster Eventing.
+
+6. Apply your emitter Function:
+
+    ```bash
+    kyma apply function
+    ```
+   Having applied it, your Function is built and deployed in Kyma runtime. Kyma will expose it via API Rule. Any incoming payload would be processed by your emitter Function. It sends the sanitised content to whatever workload that subscribes to selected event type - in our case - receiver Function.
+
+7. Test the first Function. Send a paylod and see if your HTTP traffic is accepted:
+
+      ```bash
+      export KYMA_DOMAIN=.... # export the variable for you Kyma domain
+      
+      curl -X POST https://incoming.${KYMA_DOMAIN}
+      -H 'Content-Type: application/json'
+      -d '{"foo":"bar"}'
+      ```
+
```

Comment:
Consider adding subheadings that will break the wall of text and make the whole tutorial more user-friendly. For example: 

```
### Create the emitter Function

steps

### Create the receiver Function

steps

````